crates/framework/src/natives/mod.rs

```diff
@@ -62,6 +81,11 @@ pub fn all_natives(gas_params: GasParameters) -> NativeFunctionTable {
     );
     add_natives!("rlp", mos_stdlib::rlp::make_all(gas_params.rlp));
     add_natives!("bcd", mos_stdlib::bcd::make_all(gas_params.bcd));
+    add_natives!(
+        "tx_context",
+        mos_stdlib::tx_context::make_all(gas_params.tx_context)
+    );
+    add_natives!("object", mos_stdlib::object::make_all(gas_params.object));
```

Comment:
If the natives function is ordered, will it be convenient for developers to check and read?